### DISCUSSION TITLE
[Help!] My account was suspended!

### DISCUSSION THREAD

**Discussion by ruoyuTz** [community/community#117507] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Dear all,

Hello! I am a newcomer to using Github. Two days ago, my account (username: TongZhao1030) was suspended, and as of now, it has not been resolved. I want to clarify that I am more than willing to comply with Github's policies. If I have violated any policies, it was unintentional (perhaps due to switching VPNs while using the platform). I am sincerely sorry for any inconvenience caused. I kindly request an opportunity to correct my mistake, and I assure you that I will not repeat it in the future. The code in my Github repository is not backed up, so it is extremely important to me.
Thank you very much for your understanding. I have asked help through Github Support and my Ticket ID is 2713785.

Best regards

**Comment by ruoyuTz** [comment]
@queenofcorgis @ghostinhershell @dmilburn

Hello there! I'm really sorry to bother you, but it's been almost a week since I contacted GitHub support, and I still haven't received a response. The issue remains unresolved, and my account is still suspended. I'm extremely anxious now. I can't even sleep properly at night. My account is really important to me, and I urgently need to use it. Could you please check the status of this ticket for me, or perhaps connect me with the relevant staff? I would greatly appreciate it, thank you so much!

**Reply by queenofcorgis** [reply]
Hi there,

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our Code of Conduct for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

**Comment by queenofcorgis** [comment, marked as answer]
The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
